default_files/grafana/30_dragonite_dimensioning_overview.json

```diff
@@ -1,8 +1,8 @@
 {
   "__inputs": [
     {
-      "name": "DS_DRAGONITE",
-      "label": "dragonite",
+      "name": "DS_MYSQL",
+      "label": "mysql",
```

Comment:
this changed the label for the datasource, on purpose?

Reply:
normally with importing it was possible to overwrite the template.. this time not, I guess because of the datasource name/label change

Reply:
are you really trying to overwrite the original **Blissey** template with the new one? Or you have some other obscure version in grafana :)